templates/ddd-service-native-db/.env.example

```
@@ -1,33 +1,53 @@
+# --- DB_BACKEND=postgresql only -----------------------------------------------------------
 DB_SCHEMA=public
 
-# Oracle only — service name replaces DB_NAME
+# --- DB_BACKEND=oracle only ---------------------------------------------------------------
+# Service name replaces DB_NAME.
 DB_SERVICE=XEPDB1
 
-# SQL Server only — ODBC driver name used to build the pyodbc connection string
+# --- DB_BACKEND=mssql only ------------------------------------------------------------------
+# ODBC driver name used to build the pyodbc connection string.
 DB_ODBC_DRIVER=ODBC Driver 17 for SQL Server
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)